### DISCUSSION TITLE
How to remove notification icon (glucose reading + graph + delta) in notification area (top of the phone screen)?

### DISCUSSION THREAD

**Discussion by bkuz114** [NightscoutFoundation/xDrip#3485] [Q&A] [answered]
For context, I have a motorola e6, Android 9, and am using Freestyle Libre3.

There is an xDrip+ icon in the notification area at the top of my phone, which displays my current glucose reading, delta, and a graph. How can I turn this notification off?

As far as I'm aware, i have disabled this feature (settings --> less common settings --> other misc. options --> unchecked "Run collector in foreground".) But it still displays. I'm assuming I just haven't found the correct feature to toggle off... can anyone help me find it please? I would prefer not to have this notification.

The only way I know how to get rid of it is to either (a) turn off all notifications for xDrip, or (b) keep my phone on DND and revoke xDrip's permission to override Do Not Disturb, However, neither scenario will work, because I need my custom low blood sugar alarm to go off when I am sleeping, while my phone is on DND (this is the entire reason I need xDrip+)

TIA for any help..

**Comment by Navid200** [comment, marked as answer]
Hope this helps:  https://github.com/NightscoutFoundation/xDrip/issues/750#issuecomment-458932675

**Comment by bkuz114** [comment]
Perfect , this is exactly the sort of thing I needed! Thank you so much. By the way - can I "close with comment", or is that something that only you guys are supposed to do? Let me know, and I can close it.

**Reply by Navid200** [reply]
There is no benefit, in my opinion, in closing discussions.
They are discussions.  Regardless of what the conclusion may be, the discussion remains valid.  By having some open and some closed, people will need to search twice to see all of them.

But, people close them.  I cannot do anything about it.

**Reply by bkuz114** [reply]
Thanks for the perspective. I will keep the discussion open, and will do so with any I end up opening here. I feel like on github there can be a pressure to close topics, so wanted to make sure I respected whatever is the approach here. But a discussion you're right is a whole different thing (not like it is a bug report). Also, maybe someone will stumble on a discussion a year down the road and want to add more information.

Thank you again for the info, and for the work put into this project. It is life changing for me, as I so often get bad lows in my sleep, and I live alone, so having a way to get woken up changes everything.

**Reply by Navid200** [reply]
Thanks,

There are different views on this.

Originally, there was no way to close discussions on GitHub.
There was a request to add one.  GitHub eventually added it.

Now, there is a different concern.
Some mods are concerned that since they want their discussions to be for questions and answers, a user may accidentally close a discussion instead of marking it as an answer.  So, they have requested for the option to close a discussion to be limited to the maintainers only.

The fact is that there is a filer.  So, someone who performs a search, can always set the filter to all.  Then, all the discussion, open or closed, will be included in their search.
The problem with that is that the filter is always set by default to open when you open discussions.
And how many people even know or care that there are filters?

May be they should change the filter default to all.  I will be happy with that.

Thanks for your contribution to the discussions.

**Reply by bkuz114** [reply]
Interesting. I was unaware of the history behind this. Yes, it seems like defaulting to all for searches would be the ideal solution. In fact, I'm not sure I understand the rationale to defaulting to open only; I suppose you're seeing what would be active (the most relevant?) discussions, but it seems that if someone is searching for a question, they'd be just as interested to see the closed/concluded discussions as well. I'm sure there's probably some good reason why it's set up the way it is that I am not considering though.
